packages/frontend/src/types/Events.ts

```diff
@@ -113,7 +113,12 @@ export enum EventName {
     COMMENTS_CLICKED = 'comments.clicked',
     NOTIFICATIONS_COMMENTS_ITEM_CLICKED = 'notifications_comments_item.clicked',
     DASHBOARD_AUTO_REFRESH_UPDATED = 'dashboard_auto_refresh.updated',
+
+    // Metrics Catalog
     METRICS_CATALOG_CHART_USAGE_CLICKED = 'metrics_catalog_chart_usage.clicked',
     METRICS_CATALOG_CHART_USAGE_CHART_CLICKED = 'metrics_catalog_chart_usage_chart.clicked',
     METRICS_CATALOG_EXPLORE_CLICKED = 'metrics_catalog_explore.clicked',
+    // TODO: Rename from tag to category
+    METRICS_CATALOG_TAG_ADDED = 'metrics_catalog_category_added',
+    METRICS_CATALOG_TAG_FILTER_APPLIED = 'metrics_catalog_category_filter_applied',
```

Comment:
Shouldn't these have `.` instead of `_` to separate the `metrics_catalog_category` action?

```suggestion
    METRICS_CATALOG_TAG_ADDED = 'metrics_catalog_category.added',
    METRICS_CATALOG_TAG_FILTER_APPLIED = 'metrics_catalog_category.filter_applied',
```

Reply:
+ I think we should perhaps also track when a user removes a category and when a tag is deleted from the project

Reply:
> I think we should perhaps also track when a user removes a category and when a tag is deleted from the project

good point! After a discussion with @PriPatel it seems that this is not an immediate need 👍

Reply:
https://github.com/lightdash/lightdash/pull/12295/commits/1955830a27ab9e0bb00405f1d56534c60553b448

renamed here!